raiden/claim.py

```diff
@@ -0,0 +1,222 @@
+import json
+from dataclasses import dataclass
+from pathlib import Path
+
+from eth_abi import encode_single
+from eth_utils import keccak, to_canonical_address, to_hex
+from web3 import Web3
+
+from raiden.settings import MediationFeeConfig
+from raiden.storage.serialization import DictSerializer
+from raiden.transfer.architecture import StateChange
+from raiden.transfer.identifiers import CanonicalIdentifier
+from raiden.transfer.mediated_transfer.mediation_fee import FeeScheduleState
+from raiden.transfer.state import (
+    NettingChannelEndState,
+    NettingChannelState,
+    SuccessfulTransactionState,
+    TokenNetworkGraphState,
+    TokenNetworkState,
+    TransactionChannelDeposit,
+)
+from raiden.transfer.state_change import (
+    ContractReceiveChannelDeposit,
+    ContractReceiveChannelNew,
+    ContractReceiveNewTokenNetwork,
+    ContractReceiveRouteNew,
+)
+from raiden.utils.formatting import to_checksum_address, to_hex_address
+from raiden.utils.signer import Signer
+from raiden.utils.typing import (
+    Address,
+    Any,
+    Balance,
+    BlockHash,
+    BlockNumber,
+    BlockTimeout,
+    Dict,
+    List,
+    Optional,
+    TokenAddress,
+    TokenNetworkAddress,
+    TokenNetworkRegistryAddress,
+    TransactionHash,
+)
+from raiden_contracts.utils.type_aliases import ChainID, ChannelID, Signature, TokenAmount
+
+CLAIM_FILE_PATH = Path("./claims.json")
+DEFAULT_SETTLE_TIMEOUT = BlockTimeout(100)
+DEFAULT_REVEAL_TIMEOUT = BlockTimeout(50)
+TOKEN_ADDRESS = TokenAddress(to_canonical_address("0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"))
```

Comment:
How do we make sure we keep track of such short-cut constants?

Reply:
I'll create an issue.

My idea is that we shouldn't create token networks in the end, but for that we need to do some more work.